### PR TITLE
fix(install): handle symlinks to capsule in workspace

### PR DESCRIPTION
**Pull request by imsnif**
## Proposed Changes

-
-
-

---
This change is [Reviewable](https://reviewable.io/reviews/teambit/bit/2445)

### CODE REVIEW

File: src/extensions/install/install.cmd.tsx [hunk 1]
```diff
@@ -21,7 +21,12 @@ export default class InstallCmd implements Command {
   // TODO: remove this ts-ignore
   // @ts-ignore
   async render() {
-    const results = await this.install.install();
-    return <Color green>Successfully installed {results.length} component(s)</Color>;
+    try {
+      const results = await this.install.install();
+      return <Color green>Successfully installed {results.length} component(s)</Color>;
+    } catch (e) {
```
Comment (GiladShoham) [inline src/extensions/install/install.cmd.tsx]: don't handle it yourself here, throw a `PaperError`

File: src/extensions/install/install.ts [hunk 2]
```diff
@@ -9,26 +8,43 @@ import { DEFAULT_REGISTRY_DOMAIN_PREFIX } from '../../constants';
...
+        const componentPackageName = componentIdToPackageName(
+          e.component.id.legacyComponentId,
+          DEFAULT_REGISTRY_DOMAIN_PREFIX
+        );
+        try {
+          await fs.unlink(path.join(process.cwd(), 'node_modules', componentPackageName));
+        } catch (err) {
+          // if the symlink does not exist - no problem
+          if (err.code !== 'ENOENT') {
+            throw err;
```
Comment (GiladShoham) [inline src/extensions/install/install.ts]: faster to check if exist instead of try catch (the catching is much slower then checking existence)

Comment (imsnif) [reply src/extensions/install/install.ts]: Checking existence is not atomic though...

File: src/extensions/install/install.cmd.tsx (hunk 1 above)
Comment (imsnif) [reply src/extensions/install/install.cmd.tsx]: I agree. I had issues with it though so did this temporarily. Will fix in this or a future PR in the next few days as I refactor things

**Pull request merged**